setup.py

```diff
@@ -43,7 +43,7 @@
         ],
         "dev": [
             "nomic[all]",
-            "black",
+            "black==24.3.0",
```

Comment:
what's the reason for pinning this?

Reply:
there was a version bump and it changed some files that I didn't modify - may make it more difficult to review the actual changes for a more complex PR. Also thought it would be nice to match the version in atlas cloud but fine with keeping it unpinned

Reply:
ah, if its pinned in atlas_cloud we can pin it here, mostly I'm worried about forgetting we've pinned it